content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -146,6 +146,11 @@ The following plugins support `WaitForFirstConsumer` with dynamic provisioning:
 * [GCEPersistentDisk](#gce-pd)
 * [AzureDisk](#azure-disk)
 
+{{< feature-state state="beta" for_k8s_version="1.14" >}}
+* CSI - see specific CSI driver's documentation for supported topology keys and
```

Comment:
```suggestion
* {{< glossary_tooltip text="CSI" term_id="csi" >}} - You'll need to look at the documentation for a specific CSI driver to see its supported topology keys and
```

This will turn “CSI” into a link to the glossary term, with a short explanation available via HTML `title` (eg as a tooltip).
I'm suggesting the revised wording because I think it's a more appropriate tone.